docs/usage/configuration-options.md

```diff
@@ -1489,6 +1489,8 @@ Use this field to restrict rules to a particular branch. e.g.
 }
 ```
 
+This field also supports Regular Expressions if they begin and end with `/`.
```

Comment:
Do we want to add an example like this one below this line?
I've grabbed the regex from the `description` field in the `lib/config/options/index.ts` file in this PR.

```json
{
  "packageRules": [
    {
      "matchBaseBranches": ["main"],
      "excludePackagePatterns": ["/^release\\/.*/"],
      "enabled": false
    }
  ]
}
```

Reply:
I had explicitly not done this, to not make documentation unnecessarily long. Bus yes I can add this. 
I can also make one documentatie block with both options:
```
Use this field to restrict rules to a particular branch. This field also supports Regular Expressions if they begin and end with `/`. e.g.

```json
{
  "packageRules": [
    {
      "matchBaseBranches": [
             "main"
             "/^release\\/.*/"
      ],
      "excludePackagePatterns": ["^eslint"],
      "enabled": false
    }
  ]
}
```
```

Reply:
I think having it in one block can be confusing.
JSON doesn't allow comments, so you cannot put a comment behind each example saying "this is regex" and " this is normal stuff".

I would like 2 separate examples, even if it is a bit longer. 😄